t/17-labels_carry_over.t

```diff
@@ -0,0 +1,110 @@
+# Copyright (C) 2016 SUSE Linux GmbH
```

Comment:
SUSE LLC is the one we should use for new code. SUSE Linux GmbH only exists to pay our bills